charts/spire/charts/spire-server/values.yaml

```diff
@@ -370,6 +370,12 @@ notifier:
 controllerManager:
   ## @param controllerManager.enabled Flag to enable controller manager
   enabled: false
+
+  ## @param controllerManager.className specify to use an explicit class name. If empty, it will be automatically set to Release.Namespace-Release.Name to not conflict with other installs, enabling parallel installs.
+  className: ""
```

Comment:
instanceName would be a better name IMHO. className has a completely different meaning to me, while so far I see this is used as the name of a spire deployment instance.

Reply:
too much history I think to change, and instanceName is not clear either, as your putting it on an instance of a CR. So, your naming the CR a second time?

Reply:
I guess I completely lack the context on these changes, I am completely lost.

Reply:
@marcofranssen The context is this pr in controller manager: https://github.com/spiffe/spire-controller-manager/pull/230